tests/src/test/scala/com/softwaremill/sttp/StreamingTests.scala

```diff
@@ -131,9 +134,70 @@ class StreamingTests
     }
   }
 
+  def monixAsyncHttpClientStreamingTests(): Unit = {
```

Comment:
maybe the tests could be parametrized with a `SttpHandler[Task, Observable[ByteBuffer]]` and same code used for both the async-http and okhttp handlers?